scss/base/generic.scss

```diff
@@ -41,3 +41,8 @@ input[type="radio"],
 input[type="checkbox"] {
   outline: 0;
 }
+
+// Hides the default <input type="file"> upload button. A <label> targeting the id of the now hidden input is required.
+input[type="file"] {
+  display: none;
```

Comment:
this will break the ability for keyboard users and screen readers to find this input.  `display: none` should never be used to hide elements that are still meant to be accessible to people.

instead use something like:
```css
opacity: 0; /* visually hides it */
position: absolute; /* to remove it from the document flow */
```

more information here:
https://scottaohara.github.io/a11y_styled_form_controls/src/file-upload/

Reply:
Thanks for your comment. 😆 

How about this? 🤔 
```html
<label class="nes-btn" for="file_upload">
  <span>Select your file</span>
  <input type="file" id="file_upload">
</label>
```
```scss
.nes-btn {
  position: relative;

  > input[type="file"] {
    position: absolute;
    opacity: 0;
  }
}
```
<img width="549" alt="スクリーンショット 2019-04-29 12 48 47" src="https://user-images.githubusercontent.com/5305599/56875648-25d9d580-6a7d-11e9-800f-9c039003bcf5.png">